install/sensors.csv

```diff
@@ -39,6 +39,7 @@ City Technology,T3H H2S sensor,01.44865262 112,NA002,06,0,,0,0,0,0,0,0,2024-03-2
 City Technology,T3STF SO2 sensor,01.45929880 013,TO003,13,0,,0,0,0,0,0,0,2024-03-01T00:20:00Z,9999-01-01T00:00:00Z
 City Technology,T3STF SO2 sensor,01.45929894 013,NA002,05,0,,0,0,0,0,0,0,2024-03-21T02:10:00Z,9999-01-01T00:00:00Z
 Danish Meteorological Institute,FGE,S0257-E0299,SBAM,50,0,,0,0,0,0,0,0,2009-03-02T00:00:00Z,2023-10-22T23:59:00Z
+Danish Meteorological Institute,FGE,S0257-E0299,EY2M,50,0,,0,0,0,0,0,0,2024-04-04T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
please update start time to be consistent with other entries in install files
`2024-04-04T16:00:00Z`

Reply:
Amended